fury/actor.py

```diff
@@ -3954,3 +3956,101 @@ def uncertainty_cone(
     angles = main_dir_uncertainty(evals, evecs, signal, sigma, b_matrix)
 
     return double_cone(centers, evecs, angles, colors, scales, opacity)
+
+
+def texture_on_cube(negx, negy, negz, posx, posy, posz, centers=(0, 0, 0)):
+    """Map RGB or RGBA textures on a cube
+
+    Parameters
+    ----------
+    negx : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    negy : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    negz : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    posx : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    posy : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    posz : ndarray
+        Input 2D RGB or RGBA array. Dtype should be uint8.
+    centers : tuple (3,)
+        The X, Y and Z coordinate of the cube, optional.
+
+         |----|
+         | +Y |
+    |----|----|----|----|
+    | -X | +Z | +X | -Z |
+    |----|----|----|----|
+         | -Y |
+         |----|
+
+    Returns
+    -------
+    actors : list[Actor]
+        A list of Actor objects, one for each face of the cube, in order.
+
+    Implementation
+    --------------
+    Check docs/examples/viz_play_cube.py
```

Comment:
to remove

Reply:
I added this because I saw `actor.py/texture_update` using the same `Implementation` tag. Removed.